src/index.js

```diff
@@ -340,7 +355,28 @@ export function ConfirmMultisigAddress({keystore, network, bip32Path, multisig,
   }
 }
 
+/**
+ * Return an interaction class for confirming a multisig addreess with
+ * the given `keystore`.
+ *
+ * @param {KEYSTORE} options.keystore - keystore to use
+ * @param {string} jsonConfig - asdf
+ * @returns {ColdcardConfig|UnsupportedInteraction} - text in shape of coldcard multisig wallet import
+ */
+export function translateConfig({keystore, jsonConfig}) {
```

Comment:
If we want to put config -> electrum or other coordinators, this won't make sense.
But then again, if there are a class of keystores that have config files, this will make sense.
I guess this works.